app/assets/images/icon-cross.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.3.3 (12072) - http://www.bohemiancoding.com/sketch -->
+    <title>icon-cross</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
+        <g id="icon-cross" sketch:type="MSArtboardGroup" fill="#FFFFFF">
+            <path d="M10.001768,5 L5.00176803,0 L0,5 L5.00176803,10 L0,15 L5.00176803,20 L10.001768,15 L15.001768,20 L20.001768,15 L15.001768,10 L20.001768,5 L15.001768,0 L10.001768,5 Z" id="Imported-Layers-Copy" sketch:type="MSShapeGroup"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
All these SVGs could do with optimising – could you run them through https://jakearchibald.github.io/svgomg/?